cilium/cmd/encrypt_status.go

```diff
@@ -88,14 +91,40 @@ func getXfrmStats() (int, map[string]int) {
 func countUniqueIpsecKeys() int {
 	// trying to mimic set type data structure
 	// using void data type as struct{} because it does not use any memory
-	set := make(map[string]void)
+	keys := make(map[string]void)
 	xfrmStates, _ := netlink.XfrmStateList(0)
 	for _, v := range xfrmStates {
-		set[string(v.Crypt.Key)] = voidType
+		keys[string(v.Crypt.Key)] = voidType
 	}
-	return len(set)
+	return len(keys)
+}
+
+func maxSequenceNumber() string {
+	var maxSeqNum string
+	out, _ := exec.Command("ip", "xfrm", "state").Output()
+	commandOutput := string(out)
+	lines := strings.Split(commandOutput, "\n")
+	regex, _ := regexp.Compile("oseq[[:blank:]](0[xX][[:xdigit:]]+)?")
```

Comment:
Please, use `regexp.MustCompile` if you know that the regexp is valid. Also, this could be extracted to a global that is evaluated only once at init.